<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OscilloscopeApp</class>
 <widget class="QMainWindow" name="OscilloscopeApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI Osciloscopio</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="load_button">
      <property name="text">
       <string>Cargar CSV</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="grid_button">
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>Mostrar Grilla</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="plot_widget" native="true">
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="controls_frame">
      <layout class="QFormLayout" name="controls_layout"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
